rootfs/Dockerfile

```diff
@@ -13,7 +14,7 @@ RUN apt-get update \
 		python-dev \
 		--no-install-recommends \
 	&& curl -sSL https://bootstrap.pypa.io/get-pip.py | python - pip==8.1.1 \
-	&& pip install --disable-pip-version-check --no-cache-dir docker-py==1.8.1 \
+	&& pip install --disable-pip-version-check --no-cache-dir git+https://github.com/kmala/docker-py.git@56016b806cd730d3de74712c161f57208e7473f7 \
```

Comment:
this will be changed before merging once https://github.com/docker/docker-py/pull/1133 is merged.